.travis.yml

```diff
@@ -2,15 +2,16 @@ language: scala
 sudo: required
 jdk:
   - oraclejdk8
+addons:
+  apt:
+    packages:
+      - oracle-java8-set-default
```

Comment:
Here  is the link of the doc https://www.peterbe.com/plog/elasticsearch-5-in-travis-ci .